e2e/cypress/tests/integration/integrations/incoming_webhook/inapp_username_profile_override_spec.js

```diff
@@ -150,8 +150,8 @@ function verifyLastPost(owner, username, iconUrl) {
     cy.get('@usernameForPopover').click();
     verifyProfilePopover(owner, username, iconUrl);
 
-    // # Click away to close propfile popover
-    cy.get('body').click();
+    // # Press escape key to close profile popover
+    cy.get('body').type('{esc}', {force: true});
```

Comment:
```suggestion
    cy.get('body').typeWithForce('{esc}');
```

Reply:
Oops, right, I missed this. Let's have it then in a separate PR to clean up most with the same pattern.

Reply:
PR submitted - https://github.com/mattermost/mattermost-webapp/pull/10778